docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -52,12 +52,11 @@ components:
     container:
       image: golang
       memoryLimit: 512Mi
-      command: ['sleep', 'infinity']
```

Comment:
Since we never mention `sleep` and `infinity` commands anywhere else in our devfile docs, I thought this commands were no longer pertinent. Does the rest of the example still make sense? Or will we need to create an entirely new source.yaml for an example?

Reply:
sleep infinity is here since golang by default ends with 0 exit code, just after it's started.
It will lead to workspace failure in Che case, not sure about ODO.
So, I would say it's a question to other places why they don't have sleep infinity, rather than why this does have.

Reply:
Okay, I re-included the sleep and infinity commands into the original two source code examples that included them.